lib/railsful/serializer.rb

```diff
@@ -49,12 +49,34 @@ def render(options)
     end
 
     # Find the right serializer for given object.
+    # First we will look if the options hash includes a serializer. If not,
+    # we try to guess the right serializer from the model/class name.
     #
-    # @param [ApplicationRecord, ActiveRecord::Relation]
+    # @param renderable [ApplicationRecord, ActiveRecord::Relation]
+    # @param options [Hash]
     # @return [Class] The serializer class.
     #
     # :reek:UtilityFunction
-    def serializer_for(renderable)
+    def serializer_for(renderable, options = {})
+      serializer_by_options(options) || serializer_by_renderable(renderable)
+    end
+
+    # Check the options hash for a serializer key.
+    #
+    # @return [Class] The serializer class.
+    #
+    # :reek:UtilityFunction
+    def serializer_by_options(options)
+      serializer = options[:serializer]
+      return unless serializer
+
+      "#{serializer.to_s.classify}Serializer".safe_constantize
```

Comment:
Would be great to have a handled case, when the given serializer is already a class contant. (e.g. for namespacing etc) Just something like:

```ruby
serializer = options[:serializer]
return unless serializer

return serializer if serializer.is_a? Class

"#{serializer.to_s.classify}Serializer".safe_constantize
```

Reply:
This allows users to do:

```ruby
render json: User.first, serializer: Admin::UserSerializer
```